docs/backend/tracking.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Backend Tracking
+---
+
+## Visits & Events
+
+For first-party analytics, we use the
+[`ahoy_matey` gem](https://github.com/ankane/ahoy), which tracks visits and
+events.
+
+We intentionally choose to limit what user data we track and persist, and have
+opted to follow the GDPR compliance standards
+[set by Ahoy](https://github.com/ankane/ahoy#gdpr-compliance-1). By default, we
+have configured the Ahoy library to mask IP addresses, have disabled geocode
+tracking, and do not track user cookies.
+
+### Visits
+
+Ahoy creates an `Ahoy::Visit` record for each visit that it tracks.
+
+By default, we have turned off visit tracking in the `ApplicationController`:
+
+```ruby
+skip_before_action :track_ahoy_visit
+```
+
+We currently only create visits on the server-side when they are required to be
+created by events. Visits can be re-enabled for specific controller actions if
+necessary, but this should be done so _with explict care_.
+
+We do not collect any personal user data when tracking visits. Our collected
+data is limited to the user's `id`. Each user has a unique `visitor_token`,
+while each visit to the site is marked with a unique `visit_token`.
+
+### Events
+
+Ahoy creates an `Ahoy::Event` record for each event that it tracks. If no visit
+is recorded for a user when an event is tracked, Ahoy will simeltaneously create
+an `Ahoy::Visit` for the event being tracked.
+
+Events can be tracked in a controller action on the backend, or with JavaScript
+on the frontend. Learn more about tracking events with JavaScript in our
+[frontend tracking guide](../frontend/tracking.md).
+
+When an event is tracked, it should include a `name` and a `properties` hash.
+When adding new events, be sure that the name is unique per-event, and that the
```

Comment:
I find that the sentence reads a bit odd:
```
When adding new events, be sure that the name is unique per-event, and that the
properties will help you differentiate the different events with enough
granularity.
```
Should this be:
```
When adding new events, be sure that the name is unique per-event. The
properties will help you differentiate the different events with enough
granularity.
```

Reply:
Oh, I like that better. Updated!